packages/api/src/controllers/user.ts

```diff
@@ -716,8 +720,11 @@ app.post(
         stripeCustomerId: customer.id,
       });
       res.status(201);
+      res.json(customer);
```

Comment:
Interesting that we have a log right above saying this API is deprecated. Perhaps check Loki to see if it's still being used and let's kill it if not?

Reply:
Yeah, I think it's actually redundant in the "create user" flow because it gets created separately as part of that but wasn't sure if it's used elsewhere

Reply:
@thomshutt FYI `req.stripe` doesn't come from the request. It's a config [injected on every request object](https://github.com/livepeer/livepeer-com/blob/6c8f97eb3f70ec632f4b6c0624dcdc06412b0822/packages/api/src/app-router.ts#L165) :harold_pain:
Not sure if this changes anything tho or I misunderstood.

Reply:
Ah okay, maybe we do receive it then - I'd prefer not to remove for this PR since I've tested in its current form, but will do a followup with that removed and check if the signup flow still does what we expect

Reply:
I see you're mentioning it here as well, I double checked in Loki, `create-subscription` is not getting called. Either way, this would still be a `create` call, so there are no places where it's used to get the subscription